src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -12,27 +13,41 @@ function cssClass(element: string) {
 
 type Props = {
   attachmentID: string;
+  // TODO: should this be optional?
```

Comment:
Actually, why do we need `attachmentURL` at all? The `onClickAttachment` is what we use to make opening it happen. 🤔

Reply:
We need it to render the image, unless there's another way to do that

Reply:
I think I'm forgetting the context for this, but why do we need `attachmentID` for that in the first place? No harm in needing it - we always have one generated consumer-side now - just wondering.

Reply:
Don't let this question block you IMO!

Reply:
Oh no, we need URL for rendering the image. The id is used throughout MessagingAttachment.tsx

Reply:
It seems like `attachmentID` could be removed if the parameters `onClickAttachment` and `onRemoveAttachment` didn't took in a function that didnt require the `attachmentID` as a parameter. That seems very possible since the consumer is passing in those functions, so the consumer could just pass the `attachmentID` to those functions so then those functions would each be of type `() => void`

Reply:
Also can the comment about the URL potentially being optional be removed?

Reply:
I think I'm gonna leave both for now

Reply:
Makes sense, especially since changing the functions of the existing props would require updating the existing use cases of it in fampo/launchpad